src/feed/type.ts

```diff
@@ -1,5 +1,6 @@
 const feedTypes = ['sequence', 'epoch'] as const
 export type FeedType = typeof feedTypes[number]
+export const DEFAULT_FEED_TYPE = 'sequence' as FeedType
```

Comment:
Do not use `as` keywords if it is not necesarry
```suggestion
export const DEFAULT_FEED_TYPE: FeedType = 'sequence'
```